Documentation/metrics.md

```diff
@@ -46,3 +82,34 @@ Abnormally high fsync duration (`fsync_durations_microseconds`) indicates disk i
 | snapshot_save_total_durations_microseconds | The total latency distributions of save called by snapshot | Summary |
 
 Abnormally high snapshot duration (`snapshot_save_total_durations_microseconds`) indicates disk issues and might cause the cluster to be unstable.
+
+
+### proxy
+
+Etcd members  operating in proxy mode do not do store operations. They forward all requests
```

Comment:
etcd member (extra space) operating ... we never use Etcd even if it is the first word in the sentence.